editor/assets/default_renderpipeline/builtin-pipeline.ts

```diff
@@ -1251,10 +1259,17 @@ if (rendering) {
                 camera,
                 this._configs.mobileMaxSpotLightShadowMaps,
             );
-
             return pass;
         }
-
+        public addPlanarShadowQueues(pass: rendering.BasicRenderPassBuilder,
+            camera: renderer.scene.Camera, mainLight: renderer.scene.DirectionalLight | null) {
+            pass.addQueue(QueueHint.RENDER_TRANSPARENT, 'planar-shadow')
```

Comment:
Please use `BLEND`. `RENDER_TRANSPARENT` is deprecated. `OPAQUE`, `MASK`, `BLEND` are consistent with glTF.